modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -61,21 +61,20 @@ If you just want to add or change some behavior for a few items, use the xref:ex
 
 Wildcards such as `*`,`+`,`?` may be used in element or attribute name matching.
 
-[discrete]
 === Variables
 
+[cols="1,6"]
 |===
 | Name | Summary
 
 | {$uid}
 | Results in a unique ID. For example, `'p[id:{$uid}]'`.
 |===
 
-Use `*[*]` to include all elements and all attributes. This can be very useful when used with the xref:invalid_elements[`invalid_elements`] option.
+Use `+*[*]+` to include all elements and all attributes. This can be very useful when used with the xref:invalid_elements[`invalid_elements`] option.
```

Comment:
There is a ref here that needs to include the full ref like so:
```suggestion
Use `+*[*]+` to include all elements and all attributes. This can be very useful when used with the xref:configure/content-filtering.adoc#invalid_elements[`invalid_elements`] option.
```